.github/workflows/checks.yml

```diff
@@ -43,13 +43,13 @@ jobs:
         run: go install honnef.co/go/tools/cmd/staticcheck@v0.4.2
 
       - name: Install golangci-lint
-        run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.2
+        run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.0
 
       - name: Lint
         run: make lint
 
       - name: Ensure go mod tidy runs without changes
         run: |
           go mod tidy
+          git update-index -q --really-refresh
```

Comment:
**suggestion:**

let's keep `--quiet` (vs. `-q`).

`--quiet` is more readable (in the sense that not everyone remembers `git` CLI flags by heart).

Reply:
there is no long form for `-q` (i.e. no `git update-index --quiet`):

```
$ git update-index -quiet --really-refresh
error: unknown switch 'u'
usage: git update-index [<options>] [--] [<file>...]

    -q                    continue refresh even when index needs update
    --ignore-submodules   refresh: ignore submodules
```

Reply:
🤦‍♂️